src/General/Brand/Brand.tsx

```diff
@@ -7,10 +7,19 @@ import { BrandContainer, BrandImage } from '../../Style/General/BrandStyle';
 const GlintsBlack = require('../../../assets/image/glints-logo-black.svg');
 const GlintsWhite = require('../../../assets/image/glints-logo-white.svg');
 
-class Brand extends React.Component<Props> {
-  handleRightClick = (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
-    const { rightClickURL, onContextMenu } = this.props;
+const Brand = (props: Props) => {
```

Comment:
I have been told destructuring props directly here, helps other devs know what all props are there in the component haha. Do you want to follow that way here?

Reply:
I was gonna say that I was destructuring the props first thing in the function's body. But doing it directly in the args forces devs more or less to add additional props to the destructuring as well, instead of just getting them from `props`, so this will keep things more consistent. Thanks!